.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Primer Components Workflow
+on: [push]
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Lint
+        run: npm run lint
+
+      - name: Test
+        run: npm test
+
+      - name: Publish to the npm registry
+        uses: "primer/publish@v1.0.0"
+        with:
+          args: "-- --unsafe-perm --allow-same-version"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+      - name: Deploy
```

Comment:
Can you actually remove this deploy action? We use the Now integration now :)

Reply:
Now it’s gone! Fixed in https://github.com/primer/components/pull/543/commits/ecc4a53f5cd9cb9e8da85bfe497fcf525408ee6d